src/OpenLoco/src/GameCommands/RaiseLowerLand.cpp

```diff
@@ -0,0 +1,407 @@
+#include "Audio/Audio.h"
+#include "Economy/Economy.h"
+#include "Economy/Expenditures.h"
+#include "GameCommands.h"
+#include "Localisation/FormatArguments.hpp"
+#include "Localisation/StringIds.h"
+#include "Map/RoadElement.h"
+#include "Map/SurfaceData.h"
+#include "Map/SurfaceElement.h"
+#include "Map/TileClearance.h"
+#include "Map/TileLoop.hpp"
+#include "Map/TileManager.h"
+#include "Map/TrackElement.h"
+#include "Objects/BridgeObject.h"
+#include "Objects/LandObject.h"
+#include "Objects/ObjectManager.h"
+#include "Objects/RoadObject.h"
+#include "Objects/TrackObject.h"
+#include "S5/S5.h"
+#include "SceneManager.h"
+#include <set>
+
+using namespace OpenLoco::Interop;
+using namespace OpenLoco::World;
+using namespace OpenLoco::World::SurfaceSlope;
+
+using OpenLoco::World::TileClearance::LessThanPos3;
+
+namespace OpenLoco::GameCommands
+{
+    static loco_global<uint32_t, 0x00F0013C> _F0013C;
+    static loco_global<uint32_t, 0x00F00140> _F00140;
+    static loco_global<uint16_t, 0x00F00144> _F00144;
+    static loco_global<uint16_t, 0x00F00170> _F00170;
+
+    // 0x00468651
+    // TODO: move to TileManager?
+    static uint32_t adjustSurfaceHeight(World::Pos2 pos, SmallZ targetBaseZ, uint8_t slopeFlags, std::set<World::Pos3, LessThanPos3>& removedBuildings, uint8_t flags)
```

Comment:
What would be a good destination for this as a common function for `raiseLand`, `lowerLand`, and ultimately the mountain tool? Should it just go into `TileManager`?

After this function has been moved, we can introduce separate unit files for `raiseLand` and `lowerLand`.